### PR TITLE
Upgrading to Bootstrap and Bootswatch 5.2.2 and to Bootstrap Icons 1.9.1

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: ['3.6', '3.7', '3.8', '3.9', '3.10']
```
Comment (greyli) [inline .github/workflows/build.yml:25]: This change isn't in the scope of this PR, please submit the changes to build.yml/tox.in in a separate PR.

**Review by greyli — CHANGES_REQUESTED** [review]

**Comment by greyli** [comment]
Please update the title and comment to describe the PR.

**Comment by PanderMusubi** [comment]
> Please update the title and comment to describe the PR.

Done in https://github.com/helloflask/bootstrap-flask/pull/258

**Comment by greyli** [comment]
Please remove the change in setup.py, which is already included with #258.

**Comment by PanderMusubi** [comment]
> Please remove the change in setup.py, which is already included in #258.
> 
> Also, please update the changelog.

done

**Comment by greyli** [comment]
Please update the changelog.

**Comment by PanderMusubi** [comment]
> Please update the changelog.

done

**Review by greyli — APPROVED** [review]

**Comment by greyli** [comment]
Thanks!

**Pull request merged**